feconf.py

```diff
@@ -621,13 +621,16 @@ def get_empty_ratings():
 USER_EXPLORATION_EMAILS_PREFIX = '/createhandler/notificationpreferences'
 USERNAME_CHECK_DATA_URL = '/usernamehandler/data'
 
+FOUNDATION_SITE_URL = 'http://oppiafoundation.org'
```

Comment:
This seems like it's probably the most efficient way of enabling it, so long as we feel comfortable routing through Cloudflare: https://hackernoon.com/set-up-ssl-on-github-pages-with-custom-domains-for-free-a576bdf51bc